### DISCUSSION TITLE
How to check the completeness of a download task

### DISCUSSION THREAD

**Discussion by Sunyp-IM** [iterate-ch/cyberduck#15866] [Q&A]
Dear Community,

I recently downloaded a set of data from S3 Amazon server through Cyberduck. The download stuck at about 95% progress and I quit Cyberduck. Now I want to check which of the downloaded files are incomplete and resume their download. I remembered that the download progress of the files had appeared in the "Transfer" window, but removed these contents in that window. Now all the buttons in the "Transfer" window are grey and cannot be operated (Please refer to the attached image). How should I reload all the downloaded items to the "Transfer" window, check the completeness of each file, and resume the download of those incomplete items?

Thank you and best regards.
![transfer](https://github.com/iterate-ch/cyberduck/assets/29849023/ce6842a6-b718-42c5-9e84-6365236e3c5e)

**Comment by tatze96** [comment]
Transfer items that were removed from the list within the transfers window can't be re-added to the list to resume them.

You could try transferring the remaining items by reinitiating the transfer and using the [Compare](https://docs.cyberduck.io/cyberduck/transfer/#compare) option as conflict resolution for existing files. Cyberduck will then compare the items based on checksum or modification date if the checksum isn't available and only download the items that are either newer or not present.

The *Compare* option can either be set directly in the *Existing Files* setting in the *Download* section within the Cyberduck preferences *Transfers* tab or within the overwrite prompt if that will appear during the download process if you've set the *Existing Files* setting to *Prompt*.
